### PR TITLE
Update changes from 13-03-2020 : better tuning for converter & pdfbox

### PR DESCRIPTION
**Pull request by chocoboSan**
Seems to have brought as well (from the PR) status changes that I think are already in Prod at the moment.

### CODE REVIEW

**Comment by chocoboSan** [comment]
Forgot to rebase, please ignore the comment about the prod modifications!

File: nuxeo-arender-rendition-apb/apb.yml [hunk 1]
```diff
@@ -73,13 +73,13 @@ plans:
...
     - name: arender_converter_resources
-      default: '{"limits": {"cpu": "1","memory": "6Gi"},"requests": {"cpu": "500m","memory": "4Gi"}}'
+      default: '{"limits": {"cpu": "1.5","memory": "6Gi"},"requests": {"cpu": "1","memory": "4Gi"}}'
       type: string
       title: Converter Pod resource requests and limits
       display_type: textarea
       display_group: Container Specs
     - name: arender_pdfbox_resources
-      default: '{"limits": {"cpu": "1","memory": "1Gi"},"requests": {"cpu": "1","memory": "1Gi"}}'
+      default: '{"limits": {"cpu": "1","memory": "1Gi"},"requests": {"cpu": "1","memory": "1.5Gi"}}'
```
Comment (dmetzler) [inline nuxeo-arender-rendition-apb/apb.yml]: `request` cannot be higher than limit.

File: nuxeo-arender-rendition-apb/defaults/main.yaml [hunk 2]
```diff
@@ -38,8 +38,8 @@ arender_jni_image_pullPolicy: IfNotPresent
 arender_pdfbox_image_pullPolicy: IfNotPresent
 
 arender_broker_resources: '{"limits": {"cpu": "1","memory": "1Gi"},"requests": {"cpu": "1","memory": "1Gi"}}'
-arender_converter_resources: '{"limits": {"cpu": "1","memory": "6Gi"},"requests": {"cpu": "500m","memory": "4Gi"}}'
-arender_pdfbox_resources: '{"limits": {"cpu": "1","memory": "1Gi"},"requests": {"cpu": "1","memory": "1Gi"}}'
+arender_converter_resources: '{"limits": {"cpu": "1.5","memory": "6Gi"},"requests": {"cpu": "500m","memory": "4Gi"}}'
+arender_pdfbox_resources: '{"limits": {"cpu": "1","memory": "1Gi"},"requests": {"cpu": "1","memory": "1.5Gi"}}'
```
Comment (dmetzler) [inline nuxeo-arender-rendition-apb/defaults/main.yaml]: `requests` cannot be higher than `limits`

File: nuxeo-arender-rendition-apb/apb.yml (hunk 1 above)
Comment (chocoboSan) [reply nuxeo-arender-rendition-apb/apb.yml]: I goofed on this one, sorry!

File: nuxeo-arender-rendition-apb/defaults/main.yaml (hunk 2 above)
Comment (chocoboSan) [reply nuxeo-arender-rendition-apb/defaults/main.yaml]: ditto

**Comment by chocoboSan** [comment]
Done changes. Had forgot as well to backport to main.yaml the converter CPU request increase.

Now should be all wired correctly.

**Pull request merged**